stash-cache-origin/sbin/xcache_image_init.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+if [ -f etc/grid-security/hostcert.pem ]; then
```

Comment:
Why the check? All implementations that we support need an xrd cert/key pair (even if only for the `xcache-reporter`)

Reply:
I do like the idea of having a descendant image init override but I think fix_certs should go into the supervisord startup

Reply:
Since we don't have an agreement if a cert is indeed needed, can we leave this as-is for now?
We can revise it later.

Reply:
Fine by me